packages/orbit-components/src/FormLabel/index.tsx

```diff
@@ -80,6 +81,9 @@ const FormLabel = styled(
     color: ${!filled || disabled ? theme.orbit.colorFormLabel : theme.orbit.colorFormLabelFilled};
     line-height: ${theme.orbit.lineHeightTextNormal};
     margin-bottom: ${theme.orbit.spaceXXSmall};
+    ${mq.largeMobile(css`
+      margin-left: ${theme.orbit.spaceXXSmall};
+    `)}
```

Comment:
Their request with this was to align the tooltip with the start of the input, but it was already happening… with this change, it actually became misaligned. Check the prints:
Before:
<img width="214" alt="image" src="https://user-images.githubusercontent.com/6265045/233991518-43bef957-0c59-477a-a7c6-9620434df056.png">

Now:
<img width="214" alt="image" src="https://user-images.githubusercontent.com/6265045/233991817-66544d4d-c24e-4618-809f-df742161147b.png">

So we either don't move forward with this change (my personal preference, but just personal) or we have to adjust the tooltip positioning as well

Reply:
I will update the position ;)